docs/Tutorial_Advanced_Annotation_Vectors.ipynb

```diff
@@ -0,0 +1,981 @@
+{
```

Comment:
<p>Should use <code>from stumpy import mass</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/408/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>